lgc/elfLinker/ColorExportShader.cpp

```diff
@@ -180,6 +180,26 @@ void ColorExportShader::updatePalMetadata(PalMetadata &palMetadata) {
         finalExportFormats.push_back(expFmt);
     } else {
       hasDepthExpFmtZero = false;
+      unsigned depthMask = info.location;
+      if (!m_killEnabled && m_pipelineState->getTargetInfo().getGfxIpVersion().major >= 11 &&
+          m_pipelineState->getColorExportState().alphaToCoverageEnable) {
+        for (auto &curInfo : m_exports) {
+          if (curInfo.hwColorTarget == EXP_TARGET_MRT_0 && (m_exportFormat[EXP_TARGET_MRT_0] > EXP_FORMAT_32_GR)) {
+            // Mrt0 is enabled and its alpha channel is enabled
+            depthMask |= 0x8;
+            break;
+          }
+        }
+      }
+
+      unsigned depthExpFmt = EXP_FORMAT_ZERO;
+      if (depthMask & 0x4)
+        depthExpFmt = EXP_FORMAT_32_ABGR;
+      else if (depthMask & 0x2)
+        depthExpFmt = (depthMask & 0x8) ? EXP_FORMAT_32_ABGR : EXP_FORMAT_32_GR;
+      else if (depthMask & 0x1)
+        depthExpFmt = (depthMask & 0x8) ? EXP_FORMAT_32_AR : EXP_FORMAT_32_R;
+      palMetadata.setSpiShaderZFormat(depthExpFmt);
```

Comment:
Is this not redundant with what happens in FragColorExport::generateExportInstructions? The latter method is being used by ColorExportShader...

Reply:
Yes, it is redundant. I originally wanted to clear it, but found these registers are not written into ELF when building glue shader, the final registers are from `pipelineState`. If the glue shader is from cache, here is still needed.  

Considering that we will use the new color export shader, glue shader may no longer be needed, so I didn't refine this part of the code.